.devcontainer/oncreate

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+echo "Running Azure SDK for Rust devcontainer post create script..."
+
+MSRV=$(cargo metadata --format-version=1 --no-deps  | jq '.packages[] | select(.name == "azure_core").rust_version' -r)
+
+echo "Updating toolchains..."
+rustup toolchain update \
+    $MSRV \
+    stable \
+    nightly \
+    --component "rustc" \
+    --component "rustfmt" \
+    --component "rust-std" \
+    --component "cargo" \
+    --component "clippy" \
+    --component "rust-analyzer"
+
+echo "Performing an initial build to cache dependencies"
```

Comment:
Overkill and harder to maintain. Just run `rustup show` with the PWD set to our repo and our `rust-toolchain.toml` will do all this automatically.